docs/application/dotnet/guides/multimedia/overview.md

```diff
@@ -1,9 +1,9 @@
-# Media and Camera
+# Multimedia
 
 
-The media and camera features introduce how you can use multimedia features in your application.
+Multimedia features introduce how you can use multimedia features in your application.
 
-You can use the following media and camera features in your .NET applications:
+You can use the following multimedia features in your .NET applications:
 
 -   [Media Content and Metadata](media-content-metadata.md)
 
```

Comment:
Line 34: Please drop the hyphen from “microphone-type” and rephrase it to “microphone type”

Reply:
done